cmd/mongocli/mongocli.go

```diff
@@ -106,5 +118,8 @@ func initConfig() {
 }
 
 func main() {
+	cobra.EnableCommandSorting = false
+	cobra.OnInitialize(initConfig)
```

Comment:
[q] I am not sure why we need these two lines

Reply:
those lines used to be in L87 and L91 I just got rid of the init function
cobra by default sorts command alphabetically, we prefer to use our own order, so `cobra.EnableCommandSorting`

Also we call `initConfig` to initialize viper